src/map/lua/lua_baseentity.h

```diff
@@ -631,6 +631,7 @@ class CLuaBaseEntity
     void  setClaimable(bool claimable);
     bool  getClaimable();
     void  clearEnmityForEntity(CLuaBaseEntity* PEntity);
+    void  clearEnmity(CLuaBaseEntity* PEntity); // clears player enmity from notoriety list
```

Comment:
This is actually clearing `EnmityContainer`, I assume this was an old comment (they have similar names but different functions)

Reply:
fixed.